_docs/03.2-build_server_plugin.markdown

```diff
@@ -110,10 +110,10 @@ exports.register.attributes = {
 
 If you are like us, and you plan on building out Your Awesome App even further, you will need to [generate a GitHup Api oAuth token](https://github.com/settings/tokens/new) to remove the preset limit for API requests. We have already added the code to accept and use the token in our Hapi server plugin above `const AUTH_TOKEN = process.env.token`.
 
-Click [here](https://github.com/settings/tokens/new) to set a new personal access token. Create your own `token description` in the form given and for `Select scopes` simply check: `public repo`. Then hit the green `Generate token` button.
+Click [here](https://github.com/settings/tokens/new) to set a new personal access token (you may have to login to your GitHub account if you are not already). Create your own `token description` in the form given and for `Select scopes` simply check: `public repo`. Then hit the green `Generate token` button.
```

Comment:
I can take it out, but if your not actively sign in to GH, there is a sign in page when you click that link. And then it takes you to your personal setting. We ask them to sign into GitHub @ the deploy step